panel/models/tabulator.ts

```diff
@@ -250,7 +291,10 @@ export class DataTabulatorView extends PanelHTMLBoxView {
           tab_column.formatter = "tickCross"
         else {
           tab_column.formatter = (cell: any) => {
-            return column.formatter.doFormat(cell.getRow(), cell, cell.getValue(), null, null)
+            const formatted = column.formatter.doFormat(cell.getRow(), cell, cell.getValue(), null, null)
+	    const node = div()
+	    node.innerHTML = formatted
+	    return node.children[0].innerHTML
```

Comment:
This completely breaks `HTMLTemplateFormatter`:

```python
pn.widgets.Tabulator(pd.DataFrame({'link': ['foo']}), 
                     formatters={'link': HTMLTemplateFormatter(
                         template='<a href="<%= value %>">Link</a>'
                     )})
```

Reply:
Thanks, can you open an issue?

Reply:
Done, #2730